src/generic/help-sidebar/index.jsx

```diff
@@ -67,22 +69,22 @@ const HelpSidebar = ({
                   </Hyperlink>
                 </li>
               )}
-              {pathname !== gradingDestination && (
+              {!gradingDestination.includes(pathname) && (
                 <li className="help-sidebar-other-link">
                   <Hyperlink rel="noopener" destination={gradingDestination}>
                     {intl.formatMessage(messages.sidebarLinkToGrading)}
                   </Hyperlink>
                 </li>
               )}
-              {pathname !== courseTeamDestination && (
+              {!courseTeamDestination.includes(pathname) && (
                 <li className="help-sidebar-other-link">
                   <Hyperlink rel="noopener" destination={courseTeamDestination}>
                     {intl.formatMessage(messages.sidebarLinkToCourseTeam)}
                   </Hyperlink>
                 </li>
               )}
               {proctoredExamSettingsUrl
-                && pathname !== proctoredExamSettingsUrl && (
+                && !proctoredExamSettingsUrl.includes(pathname) && (
                 <li className="help-sidebar-other-link">
                   <Hyperlink
                     rel="noopener"
```

Comment:
Paragon Hyperlink component has `rel="noopener noreferrer"` by default. Please remove `rel="noreferrer"` everywhere in this file

Reply:
removed everywhere